Resources/views/CRUD/base_list.html.twig

```diff
@@ -217,9 +217,10 @@ file that was distributed with this source code.
 
                 <ul class="dropdown-menu" role="menu">
                     {% for filter in admin.datagrid.filters if (filter.options['show_filter'] is same as(true) or filter.options['show_filter'] is null) %}
+                        {% set filterActive = ((filter.isActive() or filter.options['show_filter']) and not admin.isDefaultFilter(filter.formName)) %}
```

Comment:
@mmucklo This is where you can modify to how the filter. IMHO the ` admin.isDefaultFilter(filter.formName)` should also check if it's NOT also got filter.options[';show_filter'] as truthy.